ext/drreg/drreg.c

```diff
@@ -65,17 +65,49 @@
 
 #define PRE instrlist_meta_preinsert
 
-/* This should be pretty hard to exceed as there aren't this many GPRs */
+/* We add additional spill slots to cater for client spills.
+ * The number of spills should be pretty hard to exceed as there aren't
+ * that many GPRs.
+ */
 #define MAX_SPILLS (SPILL_SLOT_MAX + 8)
+/* We choose the number of available slots for spilling simds to match their
+ * theoretical max number on any platform, which is up to 32 for 64-bit on x86.
```

Comment:
Sounds that it's always 32 now, even though it's only 8 for 32-bit. Maybe remove the quantification here, WDYT?

Reply:
Done.